packages/prettier/package.json

```diff
@@ -13,7 +13,7 @@
     "url": "https://github.com/guardian/configs/issues"
   },
   "peerDependencies": {
-    "prettier": "^2.0.0"
+    "prettier": "^2.4.0"
```

Comment:
Why force consumers to upgrade? Are we using features that were added after 2.0.0?

Reply:
Yeah the new new brackets config comes with 2.4